src/style.css

```diff
@@ -2,9 +2,109 @@
 *::after,
 *::before {
   box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  --color-primary: #1a2236;
+  --color-white: #fff;
+  --font-primary: "Playfair Display", serif;
+  --font-secondary: poppins, sans-serif;
 }
 
+/* @font-face {
+  font-family: Poppins;
+  src: url("/fonts/Poppins/Poppins-Regular.ttf");
+}
+
+@font-face {
+  font-family: Playfair Display;
+  src: url("/fonts/Playfair_Display/static/PlayfairDisplay-Regular.ttf");
+} */
+
 html {
-  margin: 0;
-  padding: 0;
+  font-family: var(--font-secondary);
+  font-size: 16px;
+  font-style: normal;
+  font-weight: normal;
+}
+
+.wrapper {
+  display: flex;
+  flex-direction: column;
+  width: 100vw;
+  min-height: 100vh;
+  background-image: url("/images/image_geometry_1.svg");
+  background-repeat: no-repeat;
+  color: var(--color-primary);
+}
+
+.header {
+  display: flex;
+  justify-content: flex-end;
+}
+
+.header .menu-icon {
+  width: 20px;
+  height: 20px;
+  background-image: url("/images/ic_menu.svg");
+  background-repeat: no-repeat;
+  margin-block-start: 19px;
+  margin-inline-end: 16px;
+}
+
+.main {
+  margin-block-start: 104px;
+  padding-inline: 16px;
+}
+
+.main .title {
+  font-family: var(--font-primary);
+  font-size: 50px;
+  line-height: 100%;
+  margin-block-end: 20px;
+}
+
+.main .subtitle {
+  font-size: 14px;
+  line-height: 21px;
+  letter-spacing: -0.01em;
+  margin-block-end: 40px;
+}
+
+.main .buttons {
+  margin-block-end: 32px;
+}
+
+.main .divider {
+  background-image: url("/images/image_geometry_2.png");
+  background-size: cover;
+  height: 207px;
+  background-repeat: no-repeat;
+  background-position-x: center;
+  width: calc(100vw + 1px);
+  margin-inline: -16px;
+}
+
+.btn {
+  padding-block: 12px;
+  padding-inline: 42px;
+  background-color: var(--color-primary);
+  color: var(--color-white);
+  border: 1px solid var(--color-primary);
+  width: 100%;
+  cursor: pointer;
```

Comment:
- [ ] Great job styling this button, as well as its interactions. But for this project's requirements, you should stick to the Figma design as much as possible. Paddings and colors look perfect, but in the original design there is no border, and there is a fixed height and border, as you can see here:

![image](https://user-images.githubusercontent.com/66967274/138204809-66e9ab7d-a194-46d0-bb29-ee35fbd840a0.png)

Reply:
The reference shows a left-right margin of 16px. So the width might change dynamically to the mobile screen.